### ISSUE TITLE
develop http client (`rama-py`) for python

### ISSUE THREAD

**Issue by GlenDC** [plabayo/rama#342]
This will be tackled by me,
but only after 0.3 is released. Not a priority now,
and nor do I want it earlier.

Hope is that after 0.3 the core API is stable enough that
maintaining this bridge should be easy.

**Comment by GlenDC** [comment]
There will be two crates:

- `rama-py-core`: will contain all the core primitives and stuff that one can use to make their own python crate for their rama-based codebase
- `rama-py`: an opiniated rama python client lib that we'll ship

For the latter we probably want to use https://github.com/PyO3/maturin to make our life easy. Will need to see how easy it is to make `rama-py-core`, still on top of `pyo3` I would think, in worst case manual C FFI or something else entirely.
